src/tvheadendApi.test.js

```diff
@@ -0,0 +1,49 @@
+const test = require('ava');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
const test = require("ava");
```